subt/teambase.py

```diff
@@ -20,4 +30,27 @@ def update(self):
         if handler is not None:
             handler(getattr(self, channel))
 
+
+def main():
+    import argparse
+    import os
+    from osgar.lib.config import config_load
+    from osgar.record import record
+
+    parser = argparse.ArgumentParser(description='SubT Teambase')
+    parser.add_argument('config', nargs='+', help='configuration file')
+    parser.add_argument('--note', help='add description')
+    parser.add_argument('--log', nargs='?', help='record log filename')
+    parser.add_argument('--robot-name', required=True, help='T<simulated seconds>F* name')
+    args = parser.parse_args()
+
+    cfg = config_load(*args.config)
+    cfg['robot']['modules']['app']['init']['robot_name'] = args.robot_name
+    prefix = os.path.basename(args.config[0]).split('.')[0] + '-'
+    record(cfg, prefix, args.log)
```

Comment:
Is the log being uploaded to the bridge?

Reply:
yes, that was the reason to imitate what normal robots do, i.e. almost reuse the command line arguments